maps/away/sol_shuttle/sol_shuttle.dm

```diff
@@ -0,0 +1,102 @@
+#include "sol_shuttle_jobs.dm"
+#include "sol_shuttle_areas.dm"
+#include "sol_shuttle_radio.dm"
+
+/datum/map_template/ruin/away_site/sol_shuttle
+	name = "SolGov Shuttle"
+	id = "awaysite_sol"
+	description = "SolGov Cruiser."
+	suffixes = list("sol_shuttle/sol_shuttle.dmm")
+	area_usage_test_exempted_root_areas = list(/area/sol_shuttle)
+	spawn_cost = 0.6
+	player_cost = 4
+	spawn_weight = 0.67
+	shuttles_to_initialise = list(/datum/shuttle/autodock/overmap/sol_shuttle)
+
+/datum/shuttle/autodock/overmap/sol_shuttle
+	name = "SolGov Cruiser"
+	move_time = 10
+	shuttle_area = list(
+		/area/sol_shuttle/cryo,
+		/area/sol_shuttle/crew,
+		/area/sol_shuttle/ofd,
+		/area/sol_shuttle/science,
+		/area/sol_shuttle/bridge,
+		/area/sol_shuttle/primaryhall,
+		/area/sol_shuttle/secondaryhall,
+		/area/sol_shuttle/medbay,
+		/area/sol_shuttle/hydro,
+		/area/sol_shuttle/engineering,
+		/area/sol_shuttle/engine1,
+		/area/sol_shuttle/engine2,
+		/area/sol_shuttle/electrical,
+		/area/sol_shuttle/atmospherics,
+		/area/sol_shuttle/lounge,
+		/area/sol_shuttle/capsroom,
+		/area/sol_shuttle/telecomms,
+	)
+	dock_target = "sol_shuttle"
+	current_location = "nav_hangar_sol_shuttle"
+	landmark_transition = "nav_transit_sol_shuttle"
+	range = 1
+	fuel_consumption = 4
+	ceiling_type = /turf/simulated/floor/shuttle_ceiling
+	defer_initialisation = TRUE
+
+/obj/effect/overmap/visitable/sector/sol_shuttle_spawn
+	name = "small asteroid cluster"
+	desc = "Sensor array detects a small asteroid cluster."
+	in_space = 1
+	icon_state = "meteor2"
+	hide_from_reports = TRUE
+	initial_generic_waypoints = list(
+		"nav_sol_1"
+	)
+
+	initial_restricted_waypoints = list(
+		"SolGov Cruiser" = list("nav_hangar_sol_shuttle")
+	)
+
+/obj/effect/overmap/visitable/ship/landable/sol_shuttle
+	name = "SolGov Cruiser"
```

Comment:
Might want to give it a proper ship name, instead of simply saying it's "SolGov".
You know, the prefix, similar to "SEV" and "TGLV".